std/internal/math/gammafunction.d

```diff
@@ -1418,7 +1423,7 @@ assert(gammaIncompleteComplInv(3, 0)==real.infinity);
 // Fixed a bug that caused gammaIncompleteCompl to return a wrong value when
 // x was larger than a, but not by much, and both were large:
 // The value is from WolframAlpha (Gamma[100000, 100001, inf] / Gamma[100000])
-assert(fabs(gammaIncompleteCompl(100000, 100001) - 0.49831792109) < 0.000000000005);
+assert(fabs(gammaIncompleteCompl(100000, 100001) - 0.49831792109) < 0.00000005);
```

Comment:
Again, please do not reduce the precision of 80 bit tests. Add a separate test for 64 bit reals.